src/uikit.cs

```diff
@@ -14720,12 +14724,15 @@ partial interface UIPrinterPickerController {
 		[Static, Export ("printerPickerControllerWithInitiallySelectedPrinter:")]
 		UIPrinterPickerController FromPrinter ([NullAllowed] UIPrinter printer);
 	
+		[Async (ResultTypeName = "UIPrinterPickerCompletionResult")]
```

Comment:
right, easy to avoid (in test) but we should warn when encountered -> https://bugzilla.xamarin.com/show_bug.cgi?id=53103